src/import/allpaths.c

```diff
@@ -90,8 +90,8 @@ set_tablesample_rel_pathlist(PlannerInfo *root, RelOptInfo *rel, RangeTblEntry *
 }
 
 /* copied from allpaths.c */
-void
-ts_create_plain_partial_paths(PlannerInfo *root, RelOptInfo *rel)
+static void
+create_plain_partial_paths(PlannerInfo *root, RelOptInfo *rel)
 {
```

Comment:
This is a matter of taste, but I would still leave the ts_ prefix so that it's harder to confuse with grep/go to definition.

Reply:
Changed back to `ts_create_plain_partial_paths` as suggested.